sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -688,6 +707,9 @@ public partial class BlobItemProperties
         public string EncryptionScope { get { throw null; } }
         public Azure.ETag? ETag { get { throw null; } }
         public System.DateTimeOffset? ExpiresOn { get { throw null; } }
+        public bool HasLegalHold { get { throw null; } }
+        public System.DateTimeOffset? ImmutabilityPolicyExpiresOn { get { throw null; } }
+        public string ImmutabilityPolicyMode { get { throw null; } }
```

Comment:
Note that `ImmutabilityPolicyMode` is a string, not a `BlobImmutabilityPolicyMode`.  This is because the service can return a 3rd state, `Mutable`, which is not settable by the user.

Reply:
Do we expect service to add more values to this enum?
Maybe we could have enum here but validate input where this is being set ?
Or maybe we should have two enums?

Reply:
I have resolved this issue.  We are using a single enum, but validating it when it is being used as a parameter.